java/AxivityReader.java

```diff
@@ -16,6 +18,10 @@
  */
 public class AxivityReader extends DeviceReader {
 
+    // public static ZonedDateTime SESSION_START = null;
+    // public static ZoneId zoneId = ZoneId.of("Europe/London");
+    public static LocalDateTime SESSION_START = null;
```

Comment:
`SESSION_START` is persisted throughout the cwa file reads. This is the _only_ reliable local time that we have as any subsequent timestamp may have suffered a DST crossover.